test/prog.c

```diff
@@ -82,12 +83,11 @@ void
 prog_print_common_options (const struct prog *prog, FILE *out)
 {
     fprintf(out,
-"   -s SVCPORT  Service port.  Takes on the form of IPv4:port or\n"
-"                 IPv6:port.  For example:\n"
-"                   127.0.0.1:12345\n"
-"                   ::1:12345\n"
-"                 If no -s option is given, 0.0.0.0:12345 address\n"
+"   -s SVCPORT  The port on which the client should connect.\n"
+"                 If no -s option is given, port 443\n"
```

Comment:
I think examples should stay.  We should add an example where `-s hostname:port` is specified as well as `-s port`.